shared/cfg/guest-os/Linux/Fedora/22.x86_64.cfg

```diff
@@ -0,0 +1,22 @@
+- 22.x86_64:
```

Comment:
I'm wondering if we should add `armhfp`, but I'm not aware of anybody using it so we can probably wait for user request.